atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -226,12 +231,32 @@ def manifest_list_entries_match(self, image, build_id):
             v2_digest = archive['extra']['docker']['digests'][v2_type]
             koji_archives_data[arch] = v2_digest
 
-        if koji_archives_data == manifest_list_data:
-            self.log.info('Deeper manifest list check verified v2 manifest references match')
-            return True
-        self.log.warning('Manifest list refs "%s" do not match koji archive refs "%s"',
-                         manifest_list_data, koji_archives_data)
-        return False
+        platform_to_arch_dict = get_platform_to_goarch_mapping(self.workflow)
+        architectures = []
+        for platform in self.platforms:
+            architectures.append(platform_to_arch_dict[platform])
+
+        if not all(arch in koji_archives_data for arch in architectures):
+            self.log.warning('Requested platforms "%s" are missing in Koji archives "%s"',
```

Comment:
I think this log could be reworded to be more specific, because it does not tell whether all or which ones are missing in Koji archives data. That could probably means the call of `all` should be replaced with another way to find the difference.

Reply:
If this warning is raised it means that user requested platforms which are not presented in Koji Archive. For us it means that parent check failed and build won't be created. This message suppose to work as debug info why build failed.

I can change the check to following and print only missing arches:
```
        missing_arches = [arch for arch in architectures if arch not in koji_archives_data]
        if missing_arches:
            self.log.warning('Architectures "%s" are missing in Koji archives "%s"',
                             missing_arches, koji_archives_data)
```